### DISCUSSION TITLE
[Feature Request] add `tz=utc` parameter to commits list url to group by UTC dates

### DISCUSSION THREAD

**Discussion by andry81** [community/community#60003] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

If type the url:

https://github.com/USER/REPO/commits?branch=BRANCH&until=YYYY-MM-DD

Then the GitHub will group commits by days and start from the YYYY-MM-DD, where DD will contain UTC dated commits. The rest of commits of the day DD may be shifted to the YYYY-MM-DD+1 day because of +N local timezone.

Is there a chance to fix that by adding the respective parameter to the url or use UTC time by default?

**Comment by nirav-gajera** [comment, marked as answer]
You can add the parameter `&time_zone=UTC` to the URL to use UTC time by default. This will ensure that the commits are grouped by days and start from the specified date in UTC time zone. 

Browser Extensions: There might be browser extensions or userscripts available that modify the display of commit history on GitHub.

Git Command Line: If you have access to the Git command-line interface, you can use git log  `git log --date=iso --date-order
`
